src/settings.js

```diff
@@ -874,6 +874,11 @@ var INCOMING_MODULE_JS_API = [
   'stdin', 'stdout', 'thisProgram', 'wasm', 'wasmBinary', 'websocket'
 ];
 
+// Set of addiional symbols that can be added to `INCOMING_MODULE_JS_API`.
+// Do not modify this settings, it is used purely to document the symbols
+// available to `INCOMING_MODULE_JS_API`.
+var ADDITIONAL_INCOMING_MODULE_JS_API = ['fetchSettings'];
```

Comment:
Why make this a public setting, or a setting at all, if the docs say not to modify it? If it's for lack of a better place to document it, then we have `module.rst`. And the list can be a constant list in `emcc.py`.

Reply:
Yes, this is purely for documentation purposes.   you think have them live in emcc.py is OK?